apps/src/applab/applab.js

```diff
@@ -1682,6 +1620,16 @@ Applab.setImageURL = function (opts) {
   return false;
 };
 
+Applab.playSound = function (opts) {
+  if (studioApp.cdoSounds) {
+    studioApp.cdoSounds.playURL(opts.url,
+                               {volume: 1.0,
+                                forceHTML5: true,
```

Comment:
I didn't change this part for now because the only parameter to the student is still just the URL. These other options are embedded in the contract between Applab and the Sounds library. While we wrestle with the right way to do playSound() in Applab, we may iterate on this a little further. I think it is ok for the applab writers (code.org devs) to understand the differences between using HTML5 audio and not. If it was simply a choice of cross-domain or not, we could hide it. But it seems to be more complex (browser compat, mobile, and cross-domain).